### ISSUE TITLE
hepatocyte depolarization/polarity and valproic acid-induced hepatotoxicity not being returned by name resolver

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/Feedback#75]
Certain phenotypes do not readily map to vocabularies / CURIEs. Identifying appropriate CURIES to map to, e.g., lists of phenotypes can be arduous, in part because Name Resolver does not support many of the phenotypes provided by SMEs, e.g. hepatocyte depolarization/polarity and valproic acid-induced hepatotoxicity. Also see the QotM6 case phenotypes: https://github.com/NCATSTranslator/testing/issues/233.

**Comment by GitHubbit** [comment]
Our KP uses UMLS - UMLS is designed for this purpose, as it is a terminology crosswalk. Is there much interest in UMLS being more rigorously integrated into Translator? It has much value.

**Comment by karafecho** [comment]
I agree, @GitHubbit. UMLS is general purpose and could be very valuable to Translator. I posted this ticket, in part, to promote a broader discussion. Thanks for your input!

**Comment by gaurav** [comment]
> in part because Name Resolver does not support many of the phenotypes provided by SMEs, e.g. hepatocyte depolarization/polarity and valproic acid-induced hepatotoxicity

The RENCI NameRes (https://name-resolution-sri.renci.org/docs) should have all of UMLS in it now, and we'll be pushing this release to ITRB CI next week and then ITRB Test/Prod the following week unless something goes wrong. Could you please let me know which UMLS terms you're looking for?

**Comment by sierra-moxon** [comment]
Hi @gaurav - can we close this ticket now? :)

**Comment by gaurav** [comment]
Do we have some example UMLS terms I could query? I couldn't find "hepatocyte depolarization/polarity" or "valproic acid-induced hepatotoxicity" in UMLS, although similar terms like "hepatocyte dedifferentiation" (GO:1990828, UMLS:C4235317) and "Poisoning by, adverse effect of and underdosing of valproic acid" (UMLS:C2878193) are present in NameRes on ITRB.

Note that all of this has been deployed to RENCI as dev and ITRB as CI and Test -- I'll be requesting deployment to ITRB Prod this Friday after the Relay.

**Comment by karafecho** [comment]
I searched UMLS and also could not find the exact terms. The ones that Gaurav found are not close enough matches.

The specific issue here might be easier to troubleshoot if we had a specific QotM question(s)/query(ies) to explore, but I don't think we formulated one that was specific enough. As such, we could either: (1) see if we can identify another knowledge source that represents the terms of interest; (2) take a closer look at the QotM GitHub posts and see if we can identify an actual question(s)/query(ies); or (3) close the ticket on the assumption that we will be able to handle a question(s)/query(ies) related to "hepatocyte deplorization/polarity" and "valproic acid-induced hepatotoxicity" through the use of qualifiers and/or a newly digested knowledge source.

My vote is (3), but I defer to @sierra-moxon and @gaurav.
